contracts/interchain-token-service/src/events.rs

```diff
@@ -37,9 +38,11 @@ impl From<Event> for cosmwasm_std::Event {
             Event::TokenMetadataRegistered {
                 token_address,
                 decimals,
+                source_chain,
             } => cosmwasm_std::Event::new("token_metadata_registered")
                 .add_attribute_as_string("token_address", token_address)
-                .add_attribute_as_string("decimals", decimals),
+                .add_attribute_as_string("decimals", decimals)
+                .add_attribute_as_string("source_chain", source_chain),
             Event::ItsContractRegistered { chain, address } => {
```

Comment:
are we updating ITS hub events to use the new macro to go in the upcoming release?